.circleci/config.yml

```diff
@@ -6,6 +6,17 @@ orbs:
   git: wordpress-mobile/git@1.0
   slack: circleci/slack@3.4.2
 
+parameters:
+  beta_build:
+    type: boolean
+    default: false 
+  release_build:
+    type: boolean 
+    default: false
```

Comment:
Same Q as https://github.com/wordpress-mobile/WordPress-Android/pull/14289#discussion_r604285556 : can't we merge those 2 parameters into one?

Reply:
Yes, we could. I preferred to keep them separate because in some apps we have different steps for beta and release (e.g. on WPiOS we publish to AppCenter on beta, but not on release) and so I opted to keep the same infrastructure on every app. 
This doesn't add a lot of code to maintain (it's just a parameter. I'm usually against adding a ton of code for things that could be done in 2 lines just for the sake of making it future proof :-) ), but make all the code ready to support specific steps for the two cases.